自分のPC上
python3 seq2seq_batch.py --mode test --model_dir ../../../pytorch_data/10_09_1136mini --encoder=encoder_100.pth --decoder=decoder_100.pth

711サーバ上
time python3.6 seq2seq_attention_batch.py --mode test --model_dir ../../../pytorch_data/10_10_2344all_seq2seq --encoder encoder_74.pth --decoder decoder_74.pth

time python3.6 baseline_RNNLM_ngram.py --mode test --model_dir RNNLM10_23_1240_N5 --model_name model_16.pth --ngrams 5




-----------------------
OSError: Cannot read model 'enwikiALL_splited_N7.arpa' (util/mmap.cc:234 in void util::HugeMalloc(std::size_t, bool, util::scoped_memory&) threw ErrnoException because `!to.get()'. Cannot allocate memory Failed to allocate 221752468232 bytes Byte: 136)
